Empire/Empire.cpp

```diff
@@ -219,32 +261,20 @@ namespace {
 
             // get resource sharing group and amount of resource available to build this item
             const std::set<int>& group = queue_element_resource_sharing_object_groups[i];
-            if (group.empty()) {
-                //DebugLogger() << "resource sharing group for queue element is empty.  not allocating any resources to element";
-                queue_element.allocated_pp = 0.0f;
-                ++i;
-                continue;
-            }
-
             std::map<std::set<int>, float>::iterator available_pp_it = available_pp.find(group);
-            if (available_pp_it == available_pp.end()) {
-                // item is not being built at an object that has access to resources, so it can't be produced.
-                //DebugLogger() << "no resource sharing group for production queue element";
+            float& group_pp_available = (available_pp_it != available_pp.end()) ? 
+                        available_pp_it->second : dummy_pp_source;
```

Comment:
same issue as with the other continuation line-- if you still want it changed please specify the indentation that you want.